### PR TITLE
styling and UI elements for analysis tab

### PR DESCRIPTION
**Pull request by vaidotasp**
Part of #863
- Implements UI workflow and styling to get closer to PROD analysis tab

Out of scope:
- mobile styling
- required changes to vega chart api call
- ui elements rendering based on config (date range, density and datepicker)

### CODE REVIEW

File: src/js/components/leftPanel/analysisPanel/BaseAnalysis.tsx [hunk 1]
```diff
@@ -18,15 +20,18 @@ const AnalysisSpinner = (): React.ReactElement => (
 const BaseAnalysis = (): JSX.Element => {
   const dispatch = useDispatch();
   const [vegaSpec, setVegaSpec] = useState(null);
-  const [renderEditButton, setRenderEditButton] = useState(true);
+  const [renderEditButton, setRenderEditButton] = useState(false);
```
Comment (eyosef) [inline src/js/components/leftPanel/analysisPanel/BaseAnalysis.tsx:23]: @vaidotasp looks like this is referenced in `setActiveButton()` but `setActiveButton()` isn't being used. What are your thoughts on nixing this and `setActiveButton()`?

File: src/js/components/leftPanel/analysisPanel/BaseAnalysis.tsx [hunk 2]
```diff
@@ -82,20 +87,76 @@ const BaseAnalysis = (): JSX.Element => {
...
-          </option>
-        );
-      })}
-    </select>
-  );
+  const AnalysisInstructions = (): JSX.Element | null => {
+    const currentAnalysis = analysisModules.find(
+      module => module.analysisId === selectedAnalysis
+    );
+    console.log(currentAnalysis);
```
Comment (eyosef) [inline src/js/components/leftPanel/analysisPanel/BaseAnalysis.tsx]: @vaidotasp there are a few `console.log()`, so I was thinking we could pair them down? If it's more beneficial to keep them, that's totally fine too. I've kept a few in previous PRs and found it to be very helpful

File: src/js/components/leftPanel/analysisPanel/analysisTranslations.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
+export default {
+  RUN_ANALYSIS_BUTTON_TEXT: {
```
Comment (eyosef) [inline src/js/components/leftPanel/analysisPanel/analysisTranslations.ts]: @vaidotasp what are your thoughts on using camel case in this file? I noticed we've used camel case in previous content config files. Also, I was thinking we could name the object `runAnalysisButton`?

**Comment by vaidotasp** [comment]
@eyosef made the requested changes, take a look when you get a chance! 
f48cadd

**Comment by eyosef** [comment]
Looks like the `useEffect()` in `BaseAnalysis.tsx` might be the source of a memory leak. Workflow is to select a polygon on the map, and then select the analysis tab. Including a screenshot to support debugging;
Screen Shot 2020-04-02 at 2 27 35 PM

**Comment by vaidotasp** [comment]
I dont think I touched useEffect on this PR. I propose creating a new issue to fix this bug.

**Comment by eyosef** [comment]
Sounds good to me @vaidotasp

**Review by eyosef — APPROVED** [review]

**Pull request merged**